js_modules/dagster-ui/packages/ui-core/src/graph/OpTags.tsx

```diff
@@ -377,6 +420,134 @@ export const KNOWN_TAGS = {
     icon: polars,
     content: 'Polars',
   },
+  catboost: {
+    color: null,
+    icon: catboost,
+    content: 'CatBoost',
+  },
+  rust: {
+    color: '#000000',
+    icon: rust,
+    content: 'Rust',
+    reversed: true,
+  },
+  pytorchlightning: {
+    color: null,
+    icon: pytorch_lightning,
+    content: 'Pytorch Lightning',
+  },
+  deltalake: {
+    color: '#00ADD4',
+    icon: delta_lake,
+    content: 'Delta Lake',
+  },
+  parquet: {
+    color: '#50ABF1',
+    icon: parquet,
+    content: 'Parquet',
+  },
+  lightgbm: {
+    color: null,
+    icon: lightgbm,
+    content: 'lightgbm',
+  },
+  xgboost: {
+    color: '#1A9EDB',
+    icon: xgboost,
+    content: 'xgboost',
+  },
+  jax: {
+    color: null,
+    icon: jax,
+    content: 'JAX',
+  },
+  rockset: {
+    color: null,
+    icon: rockset,
+    content: 'Rockset',
+  },
+  optuna: {
+    color: '#1488C9',
+    icon: optuna,
+    content: 'Optuna',
+  },
+  chalk: {
+    color: '#000000',
+    icon: chalk,
+    content: 'Chalk',
+    reversed: true,
+  },
+  excel: {
+    color: '#00A651',
+    icon: excel,
+    content: 'Excel',
+  },
+  ray: {
+    color: '#00A2E9',
+    icon: ray,
+    content: 'Ray',
+  },
+  axioma: {
+    color: '#0774B6',
+    icon: axioma,
+    content: 'Axioma',
+  },
+  cube: {
+    color: null,
+    icon: cube,
+    content: 'cube',
+  },
+  metabase: {
+    color: '#509EE3',
+    icon: metabase,
+    content: 'Metabase',
+  },
+  linear: {
+    color: '#5E6AD2',
+    icon: linear,
+    content: 'Linear',
+  },
+  notion: {
+    color: '#000000',
+    icon: notion,
+    content: 'Notion',
+    reversed: true,
+  },
+  hackernewsapi: {
+    color: '#FB651E',
+    icon: hackernewsapi,
+    content: 'Hackernews API',
```

Comment:
"Hacker News API"?

Reply:
Added it because we use this in our starter project template. This way new users start with a nice-looking asset graph out of the box.